sharepoint/docs-conceptual/sharepoint-online/connect-sharepoint-online.md

```diff
@@ -29,7 +29,7 @@ Update-Module -Name Microsoft.Online.SharePoint.PowerShell
 
 To open the SharePoint Online Management Shell command prompt, from the **Start** screen, type **sharepoint**, and then click **SharePoint Online Management Shell**.
 
-[!VIDEO https://www.youtube.com/embed/TMzHAWEQjlk]
+[!Video https://www.youtube.com/embed/TMzHAWEQjlk]
```

Comment:
I am not 100% sure but I think that the VIDEO tag is correct but needs a >
`> [!VIDEO https://...]` 

Example https://raw.githubusercontent.com/MicrosoftDocs/azure-docs/master/articles/azure-portal/azure-portal-video-series.md 

https://docs.microsoft.com/en-us/azure/azure-portal/azure-portal-video-series

Reply:
(Thanks for your contribution!)

Reply:
I was referencing this: https://github.com/SharePoint/sp-dev-docs/edit/master/docs/spfx/set-up-your-developer-tenant.md and missed `>`.

Updated file with suggested changes, Thank you!